app/controllers/admin/bulk_republishing_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Admin::BulkRepublishingController < Admin::BaseController
+  include Admin::RepublishingHelper
+
+  before_action :enforce_permissions!
+
+  def confirm_all
+    @bulk_content_type_metadata = bulk_content_type_metadata[params[:bulk_content_type].underscore.to_sym]
+    return render "admin/errors/not_found", status: :not_found unless @bulk_content_type_metadata
+
+    @republishing_event = RepublishingEvent.new
+  end
+
+  def republish_all_organisation_about_us_pages
+    bulk_content_type = :all_organisation_about_us_pages
+    bulk_content_type_enum_index = RepublishingEvent.bulk_content_types.fetch(bulk_content_type)
```

Comment:
I found `bulk_content_type` a little confusing, as I thought it was actually a `type` rather than an integer, maybe `bulk_content_type_integer` would be a diplomatic compromise?

Reply:
I've gone with key and value for now, but I'm open. It's a little confusing since `bulk_content_types` on the class returns a hash of string keys and integer values while `bulk_content_type` on an instance just returns the string key